src/com/carolinarollergirls/scoreboard/defaults/DefaultScoreBoardModel.java

```diff
@@ -595,7 +595,7 @@ public void scoreBoardChange(ScoreBoardEvent event) {
 				//If less than one minute of intermission is left and there is another period, 
 				// start the next period. Otherwise extend the previous period.
 				requestBatchStart();
-				pc.changeNumber(1);
+				pc.setNumber(ic.getNumber()+1);
```

Comment:
Are there any known issues with ic Number being wrong? I don't think we've ever depended on it before.

Do your proposed unittests already cover this?

Reply:
Okay, let's add that test over there then to keep things straight. I doubt this is all the cause of our problems, but it should fix some ones caused by operator error.